src/mongo_client.ts

```diff
@@ -571,6 +571,8 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> {
   /**
    * Create a new Db instance sharing the current socket connections.
    *
+   * Db name validation occurs at operation time
```

Comment:
I suggest we strike this, this isn't fully accurate and it doesn't seem worthwhile explaining what validation occurs where.

Reply:
That's true, I'll go ahead and remove it